lib/Server.js

```diff
@@ -270,8 +270,10 @@ class Server {
     const webpackURL = 'http://' + this.hostname + ':' + this.webpackPort;
     const hot = this.hot;
     const publicPath = hot ? (webpackURL + '/') : null;
+    const platform = this.entry.substring(this.entry.search(/\./) + 1);
```

Comment:
This feels a bit hack-ish to me. An explicit `--platform ios` command line switch would seem cleaner. And it's also how the RN packager cli works.